apps/myaccount/src/components/multi-factor-authentication/authenticators/backup-code-authenticator.tsx

```diff
@@ -97,6 +114,24 @@ export const BackupCodeAuthenticator : FunctionComponent<BackupCodeProps> = (
         getRemainingCount();
     }, []);
 
+    /**
+     * Check whether one of the additional authenticator is enabled
+     */
+    useEffect(() => {
+        if (
+            enabledAuthenticators.filter(
+                (authenticator: string) => authenticator !== backupCodeAuthenticatorName
+            ).length > 0
```

Comment:
Can we use `find` instead of `filter` here?  

Ref: https://www.freecodecamp.org/news/find-vs-filter-javascript/#:~:text=filter()%20returns%20an%20array,is%20present%20at%20the%20beginning.

Reply:
addressed with [7f1090d](https://github.com/wso2/identity-apps/pull/3793/commits/7f1090d4101ffb73416166498cf4a6100d0cc16e)